tests/Fixtures/WebhookController.php

```diff
@@ -19,7 +19,7 @@ public function noAnnotation(): void {
 	/**
 	 * Annotated action.
 	 * @since 1.0.0
-	 * @Webhook(event="pipeline")
+	 * @Webhook("pipeline", token="token")
```

Comment:
Check if we can inject env vars here, would be amazing for usability.

Reply:
Env vars aren't possible or i am to dumb. Added support for parameters though!

Reply:
Love it!